.evergreen/config.yml

```diff
@@ -528,16 +532,26 @@ axes:
         variables:
            STORAGE_ENGINE: "inmemory"
 
+  - id: dependencies
+    display_name: Dependencies
+    values:
+      - id: lowest
+        display_name: Lowest
+        variables:
+          DEPENDENCIES: "lowest"
 
 buildvariants:
 
 # Tests all PHP versions on all operating systems.
 # Only tests against latest MongoDB and ext-mongodb versions
 - matrix_name: "test-php-versions"
-  matrix_spec: {"os-php7": ["debian92-test", "ubuntu1804-arm64-test", "rhel70-test"], "php-versions": "*", "edge-versions": "latest-stable", "driver-versions": "latest-stable" }
+  matrix_spec: {"os-php7": "*", "php-versions": "*", "edge-versions": "latest-stable", "driver-versions": "latest-stable" }
   exclude_spec:
-    - { "os-php7": ["debian92-test", "rhel70-test"], "php-versions": "7.0", edge-versions: "*", "driver-versions": "*" }
-  display_name: "PHP ${php-versions}, ${os-php7}, MongoDB ${edge-versions}, ext-mongodb ${driver-versions}"
+    # rhel71-power8 fails due to not reaching pecl
+    - { "os-php7": "rhel71-power8", "php-versions": "*", edge-versions: "*", "driver-versions": "*" }
+    # rhel74-zseries doesn't start in a timely fashion - most likely missing executors
+    - { "os-php7": "rhel74-zseries", "php-versions": "*", edge-versions: "*", "driver-versions": "*" }
+  display_name: "* ${os-php7}, PHP ${php-versions}, MongoDB ${edge-versions}, ext-mongodb ${driver-versions}"
```

Comment:
Why does the display name start with an asterisk?

Should we use a consistent order with other display names? I noticed the "test-dependencies" matrix below has some of the components here in a different order.

Reply:
The asterisk is there to make sure these variants are sorted to the top of the list, as they are the ones we require for a PR to pass.

As for ordering in display names, I moved the main variant (e.g. PHP version in this case, MongoDB server version in other cases) to the front of the list. I have no objections against changing this if you prefer to have them alphabetically sorted in general.

Reply:
Oh, I see now that you're prioritizing the variable axes over the fixed ones, and in the "test-dependencies" matrix only the MongoDB version varies. That all makes sense to me.